server/server.nix

```diff
@@ -41,7 +41,56 @@
 mkDerivation {
   pname = "icepeak";
   version = "0.6.2";
```

Comment:
Drive-by: Should be `0.6.3`

Reply:
Ah, then the version number in `server/package.yaml` is out of date as well.

Reply:
True, that needs to be updated as well then.